app/src/main/java/com/wafflestudio/siksha2/compose/ui/community/CommentDetailDialog.kt

```diff
@@ -0,0 +1,117 @@
+package com.wafflestudio.siksha2.compose.ui.community
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Divider
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+
+@Composable
+fun CommentDetailDialog(
+    onDismissRequest: () -> Unit,
+    onClickReply: () -> Unit,
+    onClickReport: () -> Unit,
+    onClickCancel: () -> Unit
+) {
+    Dialog(onDismissRequest = onDismissRequest) {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .width(315.dp)
+                .height(165.dp)
+                .background(Color.White, shape = RoundedCornerShape(26.dp))
+                .border(
+                    width = 1.dp,
+                    color = Color.Gray,
+                    shape = RoundedCornerShape(26.dp)
```

Comment:
```suggestion
            horizontalAlignment = Alignment.CenterHorizontally,
            modifier = Modifier
                .width(315.dp)
                .height(165.dp)
                .clip(RoundedCornerShape(26.dp))
                .background(Color.White)
                .border(
                    width = 1.dp,
                    color = Color.Gray,
                    shape = RoundedCornerShape(26.dp)
```

Column에 clip 적용해 주면 각 칸에서 RoundedCornerShape이랑 background, color 따로 주지 않고 투명한 직사각형처럼 다뤄도 될 것 같아